packages/lib-utils/src/app/redux/index.ts

```diff
@@ -25,7 +25,14 @@ type UseReduxStoreResult = {
  * ```
  */
 export const useReduxStore = (): UseReduxStoreResult => {
-  const storeContext = useStore();
+  let storeContext: Store | null = null;
+  try {
+    // It'll always be invoked -- it just might blow up
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    storeContext = useStore();
+  } catch (e) {
+    // TODO: Handle this better...
```

Comment:
@andrewballantyne would like to add a `consoleLogger.error` log here if you're making more changes.

Reply:
Yup, definitely can